### DISCUSSION TITLE
Server notifications can be moved to the regular notifications section

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#15257] [Ideas] [answered]
I'm talking about notifications about the number of games played in the current session. All notifications that come now in the form of chat messages can also be transferred there.

**Comment by bdach** [comment, marked as answer]
I don't understand what this is saying.

**Reply by Moltanicaa** [reply]
This messages: games for this session, You played a map n-times etc.
![image](https://user-images.githubusercontent.com/59089206/138570962-3fcd3456-ea31-4bfb-9417-0bef5f7703ee.png)

**Reply by peppy** [reply]
Please avoid opening an issue like this.

The feature is not implemented yet.

If you are looking for it to be implemented, open a discussion for that. Don't open a discussion suggesting how something is implemented that isn't implemented yet.

**Reply by Moltanicaa** [reply]
I don't quite understand. It was said to open the discussion - it is open. I suggest sending notifications to the appropriate section instead of chatting with the banchobot. What issue are we talking about?
